src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/util/ErrorCodeTest.kt

```diff
@@ -7,7 +7,7 @@ import uk.gov.justice.digital.hmpps.incentivesapi.config.ErrorCode
 class ErrorCodeTest {
   @Test
   fun `error codes should all be unique`() {
-    val errorCodes = ErrorCode.values()
+    val errorCodes = ErrorCode.entries
```

Comment:
cf. https://kotlinlang.org/docs/whatsnew19.html#stable-replacement-of-the-enum-class-values-function